qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ProviderSteps.java

```diff
@@ -60,7 +60,7 @@ public void launchSystemAdministrationPage() {
     
     @Then ("system loads administration page")
     public void systemLoadsAdministrationPage() {
-    	assertTrue(textExists("Advanced Administration"));
+    	assertTrue(textExists("Administration"));
```

Comment:
isn't the text supposed to be `System Administration` ?

Reply:
when `administrationPage = homePage.goToAdministration();` is triggered,  the test expects to land on the `Administration` page even when it goies through the `System Administration` page. 

have a look-see at this function 
```
	public AdministrationPage goToAdministration() {
		clickOn(SYSTEM_ADMINISTRATION);
		return new SystemAdministrationPage(this).goToAdvancedAdministration();
	}
```

Reply:
are you able to share the an image of the that page?

Reply:
![Screenshot from 2022-01-18 09-10-48](https://user-images.githubusercontent.com/47120265/149880811-9a856263-469e-49bf-9c4d-ee4f7b801e98.png)